releasenotes/notes/tighten-pip-tuf-index-71cc1530c1ebe930.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    We replace the PyPI index with our own by default, in order to prevent
+    accidental installation of Datadog or even third-party packages from PyPI.
```

Comment:
anything under `security` will be listed as `Security Issues` in the changelog, so we need to be extra-careful with how we word them.

Could you:
* add more context here (in particular mention the `datadog-agent integration` command)
* maybe put these 2 release notes under `fixes` unless they're actual security issues. If we keep them under security issues let's double check all users understand that this isn't a serious security issue affecting their install